python3/vdebug/event.py

```diff
@@ -446,18 +446,28 @@ def run(self, args):
         # If in the breakpoint window, toggle breakpoint instead of adding/deleting one
         window_name = Dispatcher._get_window_name()
         if window_name  == self.session_handler.ui().windows.breakpoints().name:
-            lineno = vim.current.window.cursor[0]
-            line = self.session_handler.ui().windows.breakpoints().line_at(lineno - 1)
-            id = Dispatcher._get_breakpoint_id_breakpoint_window(line)
-            if not id:
-                log.Log("No breakpoint founr at current cursor position",
-                    log.Logger.DEBUG)
-                return
+            buf = vim.current.buffer
+            try:
+                (lnum1, col1) = buf.mark('<')
+                (lnum2, col2) = buf.mark('>')
+                lines = vim.eval('getline({}, {})'.format(lnum1, lnum2))
 
-            self.session_handler.dispatch_event("toggle_breakpoint", "{} {}".format(id, "toggle"))
+                # The visual marks do not unset automatically
+                vim.command(':delmarks <>')
+            except TypeError:
+                lineno = vim.current.window.cursor[0]
+                lines = [self.session_handler.ui().windows.breakpoints().line_at(lineno - 1)]
 
-            return
+            for line in lines:
+                id = Dispatcher._get_breakpoint_id_breakpoint_window(line)
+                if not id:
+                    log.Log("No breakpoint founr at current cursor position",
```

Comment:
Thanks for update documentation. Please see:
```
 $ git grep 'No breakpoint foun. at'
python3/vdebug/event.py:464:                    log.Log("No breakpoint founr at current cursor position",
python3/vdebug/event.py:813:                    log.Log("No breakpoint founr at current cursor position",
python3/vdebug/event.py:832:                    log.Log("No breakpoint found at current cursor position",
python3/vdebug/event.py:877:             log.Log("No breakpoint founr at current cursor position",
```
Could a class constant be used there instead of duplication?